### PR TITLE
Update camera.py

### PR DESCRIPTION
**Pull request by kk60503**
fix exception handling when camera isn't found so program does not crash

### CODE REVIEW

File: oresat_star_tracker/camera.py [hunk 1]
```diff
@@ -34,14 +44,10 @@ def read_image_size(self):
...
         context_path = "/sys/devices/platform/prucam/context_settings"
-        try:
-            with open(f"{context_path}/{name}", "r") as f:
-                value = int(f.read())
-        except FileNotFoundError:
-            raise CameraError(f"no sysfs attribute {name} for camera")
-        return value
+        with open(f"{context_path}/{name}", "r") as f:
+            value = int(f.read())
+            return value
```
Comment (ryanpdx) [inline oresat_star_tracker/camera.py:50]: don't return inside of a with block

**Review by ryanpdx — CHANGES_REQUESTED** [review]
An extra level of complexity is the kernel module wont be loaded until after ~20 seconds after boot as the PRUs must exist before the module is loaded.

This or the service that uses an object of this class will need to handle it the fact the camera will not be found at the start of the app, but should be found / accessible later on.

Also run black and isort to pass the workflow.

File: oresat_star_tracker/camera.py (hunk 1 above)
Comment (ThirteenFish) [reply oresat_star_tracker/camera.py:50]: Why not? It'll do the right thing, the resource will be cleaned up (the file closed) no matter how you leave the block, `return` included. Returning inside a with block is even called out in the relevant PEP as a reason why you'd want to use it.

**Comment by ryanpdx** [comment]
Closing in favor of #23

**Pull request closed without merging**